data/team.yml

```diff
@@ -43,6 +44,15 @@ teamLeaders:
       - icon: "tf-ion-social-github"
         URL: "https://github.com/loriab"
 
+    # matt
+    - image: "images/people/mattfull.jpg"
+      name: "Matthew Welborn"
+      designation: "Developer"
+      content: "Developer focused on documenation and machine learning"
```

Comment:
Haha sorry for the "documentation" bit... we should shake out of this pretty soon. Maybe "machine learning and high quality reference data"?

Reply:
Au contraire, I think documentation is super important. I considered saying "user experience", but that felt a little pretentious.